test/pkg/sysctl/Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: tag push
+
+BASE=alpine:3.5
+IMAGE=test-sysctl
+
+default: push
+
+hash: Dockerfile check.sh
```

Comment:
Is there a reason this package makefile is old-style rather than the new more compact form with git tree hash etc